framework/src/main/java/org/checkerframework/framework/qual/QualifierForLiterals.java

```diff
@@ -0,0 +1,28 @@
+package org.checkerframework.framework.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/** A meta-annotation that indicates what qualifier should be given to literals. */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.ANNOTATION_TYPE)
+public @interface QualifierForLiterals {
+    /**
+     * @return {@link LiteralKind}s for which an annotation should be implicitly added. For example,
```

Comment:
Can we replace `implicitly` (here and below) with something else, e.g. `automatically`?